projects/step-core/src/lib/components/function-link/function-link.component.ts

```diff
@@ -13,7 +14,7 @@ export class FunctionLinkComponent implements CustomComponent {
   @Input() context?: KeywordFunction;
 
   readonly noLink$ = (this._customColumnOptions?.options$ || of([])).pipe(
-    map((options) => options.includes('noEditorLink'))
+    map((options: any) => options.includes('noEditorLink'))
```

Comment:
Remove any. It's redundant here. Typescript automatically intercepts the correct type from source observable

Reply:
I was getting an error about this, now its gone, weird